lib/ansible/modules/remote_management/cisco/cisco_ucs_dns.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: cisco_ucs_dns
+short_description: Configures DNS on a Cisco UCS server.
+version_added: "2.4"
+description:
+   -  Configures DNS on a Cisco UCS server.
+options:
+    state:
+        description:
+         - if C(present), adds DNS server
```

Comment:
Start description with capital, trailing dot.
The description is not correct, "present" does not always add a DNS, it makes sure it is define. Otherwise it would not be idempotent.

Reply:
Please do this for all descriptions.

Reply:
Do we need to really explicitly mention the idempotent behaviour in the definition. Ansible module behaviour by default is `idempotent`.

Reply:
The current description is wrong, the module doesn't add the DNS server.
And no, not all modules are idempotent, so we cannot assume that the user can expect idempotent behavior. The description is easy to fix.